spec/acceptance/helpers.rb

```diff
@@ -0,0 +1,75 @@
+# Time constants in seconds
+HOUR =  60 * 60
+DAY  =  24 * HOUR
+YEAR = 365 * DAY
+
+# Execute `date` command on host with optional arguments
+# and get back a Ruby Time object
+#
+# @param [Host]            host   single Beaker::Host to run the command on
+# @param [Array<String>]   args   Array of arguments to be appended to the
+#                                 `date` command
+# @return [Time]    Ruby Time object
+def get_time_on(host, args = [])
+  arg_string = args.join(' ')
+  date = on(host, "date #{arg_string}").stdout.chomp
+  return Time.parse(date)
+end
+
+# Retrieve the CA enddate on a given host as a Ruby time object
+#
+# @param [Host]            host   single Beaker::Host to get CA enddate from
+#
+# @return [Time]    Ruby Time object
+def get_ca_enddate_time_on(host)
```

Comment:
First off, if this works, then superb, no need to change it.

However, my personal preference is to avoid shelling out unless there's a compelling reason to do so. This could be replaced with the following:

```ruby
require 'openssl'
cert = OpenSSL::X509::Certificate.new(File.read('ca_crt.pem')) # Assuming a local file, if copied from a remote host just pass the string
cert.not_after # => 2022-02-22 17:35:30 UTC
```